python/core/auto_generated/qgsclipper.sip.in

```diff
@@ -67,20 +70,41 @@ an open shape (linestring).
 %End
 
     static void trimPolygon( QPolygonF &pts, const QgsRectangle &clipRect );
+%Docstring
+Trims the given polygon to a rectangular box. Modify the given polygon.
+
+:param pts: polygon as 2D coordinates
+:param clipRect: clipping rectangle
+%End
+
+    static void trimPolygon( QgsLineString &pts, const QgsBox3d &clipRect );
+%Docstring
+Trims the given polygon to a pseudo 3D box. Modify the given polygon.
+
+:param pts: polygon as 2D or 3D coordinates
+:param clipRect: clipping 3D box
+%End
 
-    static QPolygonF clippedLine( const QgsCurve &curve, const QgsRectangle &clipExtent );
```

Comment:
This classifies as an api break -- we need to leave the original clippedLine method available in case plugins are using it

Reply:
ok, I will readd the old function prototype